README.md

```diff
@@ -1223,6 +1223,25 @@ Other Style Guides
     foo.init();
     ```
 
+  <a name="modules--multiline-imports-over-newlines"></a><a name="10.8"></a>
+  - [10.8](#modules--multiline-imports-over-newlines) Multi-line imports should be preferred over single line
```

Comment:
How do you think the text should be framed then? I'm not able to figure out the right words for this.